pyfstat/helper_functions.py

```diff
@@ -354,18 +354,26 @@ def get_covering_band(
     F0band=0.0,
     F1band=0.0,
     F2band=0.0,
-    orbitasini=0.0,
-    orbitPeriod=0.0,
-    orbitEcc=0.0,
+    maxOrbitAsini=0.0,
+    minOrbitPeriod=0.0,
+    maxOrbitEcc=0.0,
```

Comment:
Maximum `asini` and minimum `orbital period` (which implies maximum `orbital frequency`). Again, this makes sense.